tools/cummerbund/cummeRbund.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0"?>
-<tool id="cummeRbund" name="Plot CuffDiff" version="1.0.1">
-    <description>data with cummeRbund</description>
+<tool id="cummeRbund" name="cummeRbund" version="1.0.1">
+    <description>visualize CuffDiff output</description>
```

Comment:
According to http://cole-trapnell-lab.github.io/cufflinks/cuffdiff/index.html the correct name is "Cuffdiff".